sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -203,4 +203,18 @@ class CachedTableSuite extends QueryTest {
     sql("DROP TABLE refreshTable")
     Utils.deleteRecursively(tempPath)
   }
+
+  test("SPARK-10327 Cache Table is not working while subquery has alias in its project list") {
```

Comment:
@marmbrus I am not so sure about your mean, any example code ref will be appreciated.